src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```
@@ -0,0 +1,111 @@
+                        + "  FROM manufacturers WHERE is_deleted = 0");
+            while (resultSet.next()) {
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Long id = resultSet.getObject("id", Long.class);
+                Manufacturer manufacturer = new Manufacturer(id, name, country);
+                manufacturerList.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public void delete(Long id) {
+        String deleteRequest = "UPDATE manufacturers SET is_deleted = 1 where id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteStatement =
+                         connection.prepareStatement(
+                                 deleteRequest, Statement.RETURN_GENERATED_KEYS)) {
+            deleteStatement.setLong(1, id);
+            deleteStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateStatement =
+                        connection.prepareStatement(
+                                updateRequest, Statement.RETURN_GENERATED_KEYS)) {
+            updateStatement.setString(1, manufacturer.getName());
+            updateStatement.setString(2, manufacturer.getCountry());
+            updateStatement.setLong(3, manufacturer.getId());
+            updateStatement.executeUpdate();
+            return new Manufacturer();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
```

you should rethrow a custom exception
also, make an informative error message